Documents/Documents/Code/Controllers/Documents/ASCDocumentsViewController.swift

```diff
@@ -1589,33 +1589,17 @@ class ASCDocumentsViewController: ASCBaseTableViewController, UIGestureRecognize
         }
     }
 
-    func deleteFolderAction(folder: ASCFolder) {
+    func deleteArchive(cell: UITableViewCell?, folder: ASCFolder) {
         let alertController = UIAlertController(title: NSLocalizedString("Delete forever?", comment: ""), message: "", preferredStyle: .alert)
         let cancelAction = UIAlertAction(title: ASCLocalization.Common.cancel, style: .cancel, handler: nil)
 
-        var hud: MBProgressHUD?
-
-        hud?.mode = .indeterminate
-        hud?.label.text = NSLocalizedString("Deleting", comment: "Caption of the processing")
-
         let deleteAction = UIAlertAction(title: NSLocalizedString("Delete", comment: ""), style: .destructive) { _ in
-            hud = MBProgressHUD.showTopMost()
```

Comment:
Why do you remove activity?

Reply:
We have a separate removal method that uses HUD. Previously, I used to delete through a provider.